.travis.yml

```diff
@@ -390,7 +390,7 @@ jobs:
     - stage: test
       name: "powerpc port build"
       install:
-        - sudo apt-get install gcc-powerpc64le-linux-gnu libc6-dev-ppc64el-cross
+        - sudo apt-get install gcc-powerpc64le-linux-gnu g++-powerpc64le-linux-gnu libc6-dev-ppc64el-cross
```

Comment:
Is this needed?

Reply:
Without it the error is `gcc: error trying to exec ‘cc1plus’`. But again, if the Makefile target can be rewritten to skip te whole C++ part if there are no C++ files this wouldn't bee needed.